### ISSUE TITLE
Results are obtained by finding the maximum test results.

### ISSUE THREAD

**Issue by yaokl-nju** [ZhaoningYu1996/HM-GNN#1]
Hi zhaoning,

  This is a really interesting work! But I find a minor issue with the results. Specifically, there is no separate validation set and the results are obtained by finding the maximum test results. I think this evaluation procedure is incorrect. The correct evaluation procedure is to evaluate on training set or a validation set and obtain the test results with a maximum validation results.

  Can you take some time to answer my question? Thank you very much!

**Comment by ZhaoningYu1996** [comment]
Hi klyao,

We strictly follow the settings in previous baselines. (GIN, Graph-U-Net, GSN)
The reason is those Tudatasets are small. The results will be very unstable if you use validation and test set.

**Comment by yaokl-nju** [comment]
But such an evaluation procedure is equal to fit the testing set during training and applying more complex models can always get better testing results. I couldn't understand how to obtain a fair and correct evaluation by the adopted evaluation procedure.

**Comment by ZhaoningYu1996** [comment]
The comparison is fair as long as you ensure the experiment settings are identical. (e.g. number of layers)
